src/main/java/LottoGame.java

```diff
@@ -1,49 +1,63 @@
-import common.constant.Constants;
-import domain.BonusNumber;
-import domain.Lottos;
-import domain.TotalResult;
-import domain.WinningLotto;
+import domain.*;
 import view.InputView;
 import view.OutputView;
 
-import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class LottoGame {
 
-    public static int getPaidPrice() {
+    public static Money getPaidPrice() {
         OutputView.printPaidPriceRequest();
-        int paidPrice = InputView.getPaidPrice();
-        OutputView.printLottoCount(paidPrice / Constants.PRICE);
-        return paidPrice;
+        return new Money(InputView.getInput());
     }
 
-    public static Lottos getLottos(int count) {
-        Lottos lottos = new Lottos(count);
+    public static Count getManualLottoCount(Money paidPrice) {
+        OutputView.printManualLottoCountRequest();
+        return new Count(InputView.getInput(), paidPrice.getCount());
+    }
+
+    public static Lottos getLottos(Count manualLottoCount) {
+        if (!manualLottoCount.zero()) {
+            OutputView.printManualLottoRequest();
+        }
+        Lottos manualLottos = getManualLottos(manualLottoCount);
+        OutputView.printLottoCount(manualLottoCount.getCount(), manualLottoCount.getRemains());
+        Lottos autoLottos = Lottos.getAutoLottos(manualLottoCount.getRemains());
+        Lottos lottos = new Lottos(manualLottos, autoLottos);
         OutputView.printLottos(lottos.getPurchasedLottosNumbers());
         return lottos;
     }
 
-    public static WinningLotto getWinningLotto() {
+    public static Lottos getManualLottos(Count manualLottoCount) {
+        Lottos manualLottos = new Lottos(
+                Stream.generate(() -> InputView.getInput())
+                        .limit(manualLottoCount.getCount())
+                        .map(s -> LottoFactory.getLotto(new ManualLottoGenerator(s)))
+                        .collect(Collectors.toList())
+        );
+        return manualLottos;
+    }
```

Comment:
> 수동 로또 번호를 입력받을 때
컨트롤러에서 List 변수 없이 진행하려고 하니 Stream.generate()로 입력값을 반복해 받아 이를 로또로 생성하고
그렇게 생성된 로또들을 리스트로 묶어 Lottos를 만들었습니다.
이렇게 하는 건 괜찮을까요?

개인적으로는 Stream.generate() 보다는 익숙한 IntStream을 사용하고, 
반환 값을 Lottos가 아닌 List<Lotto>를 반환한 후에, Lottos를 만들때 수동 로또번호 목록인 List<Lotto>와 자동 로또 번호 목록인 List<Lotto>를 메서드 인자로 사용해보면 더 좋을 것 같아요! 

현재 코드는 Lottos를 먼저 만들어 사용해서 얻는 장점이 크게 안보이는 것 같아요!

```suggestion
    public static List<Lotto> getManualLottos(Count manualLottoCount) {
        return IntStream.range(0, manualLottoCount.getCount())
                .mapToObj(ignore -> LottoFactory.getLotto(new ManualLottoGenerator(InputView.getInput())))
                .collect(Collectors.toList());
    }
```

Reply:
Lottos로 변환함으로 써 얻을 수 있는 부분이 있다면 Lottos를 사용하는 것도 나쁘지 않다고 생각합니다 :)
다만 지금의 케이스에서는 그러한 부분들이 보이지 않아 List를 사용하는 것이 적절해보여요 :)